framework/commercelayer/index.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react'
+import { ReactNode } from 'react'
+import { CommercelayerProvider } from './provider'
+import {
+  CommerceConfig,
+  CommerceProvider as CoreCommerceProvider,
```

Comment:
CommerceProvider doesn't exist in `@commerce`, you want to import `CoreCommerceProvider` directly. Also it doesn't receive a `config` prop as passed below